### PR TITLE
Zoom in a region

### PR DESCRIPTION
**Pull request by chfw**
This PR introduces: `center` and `zoom` to geo's add_schema function.

```
   Geo()
        .add_schema(maptype="world", center=[6, 50.4], zoom=12)
        .add(
```

pyecharts user can choose a custom center on the map and zoom in:

Screenshot 2019-06-05 at 07 48 31

Without this PR, it looks like the following:

Screenshot 2019-06-05 at 07 48 58

### CODE REVIEW

File: pyecharts/charts/basic_charts/geo.py [hunk 1]
```diff
@@ -149,15 +149,21 @@ def add_schema(
         self,
         maptype: str = "china",
         is_roam: bool = True,
+        zoom: Numeric = None,
```
Comment (chenjiandongx) [inline pyecharts/charts/basic_charts/geo.py]: Optional[Numeric]

File: pyecharts/charts/basic_charts/geo.py [hunk 2]
```diff
@@ -149,15 +149,21 @@ def add_schema(
         self,
         maptype: str = "china",
         is_roam: bool = True,
+        zoom: Numeric = None,
+        center: Union[List, None] = None,
```
Comment (chenjiandongx) [inline pyecharts/charts/basic_charts/geo.py]: Optional[Sequence]

File: test/test_geo.py [hunk 3]
```diff
@@ -1,11 +1,15 @@
-from nose.tools import eq_
+from unittest.mock import patch
+
+from nose.tools import eq_, assert_in
 
 from example.commons import Faker
 from pyecharts import options as opts
 from pyecharts.charts import Geo
+from textwrap import dedent
```
Comment (chenjiandongx) [inline test/test_geo.py]: 引入是为了？

Comment (chfw) [reply test/test_geo.py]: 漏了。我要删除它

**Comment by chfw** [comment]
@chenjiandongx , see updates

File: pyecharts/charts/basic_charts/geo.py [hunk 4]
```diff
@@ -149,15 +149,21 @@ def add_schema(
         self,
         maptype: str = "china",
         is_roam: bool = True,
+        zoom: Optional[Numeric] = None,
+        center: Union[Sequence, None] = None,
```
Comment (chenjiandongx) [inline pyecharts/charts/basic_charts/geo.py]: Union[Sequence, None]  <==> Optional[Sequence]

**Pull request merged**